### ISSUE TITLE
Question to network.js - Problem when running in subdir

### ISSUE THREAD

**Issue by vsatmydynipnet** [owntracks/recorder#414]
I have 2 instances of ot-recorder for friends runnning on P 8081 and 8082  using a freebsd server. 

In Apache these are proxy mapped as 

```
################################################################################
# maximilien OWNTRACKS Recorder
################################################################################

ProxyPass        /max/ws        ws://127.0.0.1:8101/ws keepalive=on retry=60
ProxyPassReverse /max/ws        ws://127.0.0.1:8101/ws keepalive=on
ProxyPass        /max           http://127.0.0.1:8101/
ProxyPassReverse /max           http://127.0.0.1:8101/

<Location /max>
    AuthType Basic
    AuthName "Restricted Access"
    AuthUserFile /usr/local/etc/apache24/passwords/cpvpass
    Require user maximilien
</Location>

################################################################################
# bernhard OWNTRACKS Recorder
################################################################################

ProxyPass        /bernhard/ws        ws://127.0.0.1:8102/ws keepalive=on retry=60
ProxyPassReverse /bernhard/ws        ws://127.0.0.1:8102/ws keepalive=on
ProxyPass        /bernhard           http://127.0.0.1:8102/
ProxyPassReverse /bernhard           http://127.0.0.1:8102/

<Location /bernhard>
    AuthType Basic
    AuthName "Restricted Access"
    AuthUserFile /usr/local/etc/apache24/passwords/cpvpass
    Require user bernhard
</Location>
```
This worked kinda fine till V 0.8.4. Now with 0.9.2 I run into the problem that  in utils/network.js this is hardcoded:

```
 const apiUrl = new URL(`../${ useWebsocket ? "ws" : "api/0" }${endpoint === undefined ? "" : `/${endpoint}`}`, window.location);
 ```
 
 The problem:
 
 If I let it that way, the Track Point and Route Links, normally visible on the mail page works fine, but the main page shows no devices and links.
 
 If I remove the ../ the main page is correct, but the tracking and route links does not work anymore.
 
 I am no dev so unable to go further. Any Idea would be greatly appreciated.
 
 Thank you for any suggestion.

**Comment by jpmens** [comment]
Thanks for the report, which I'm labeling as a bug, but help is needed for this JavaScript issue.

**Comment by ckrey** [comment]
The problem is fixed for 0.9.3 here 4dd7a84

**Comment by jpmens** [comment]
Thanks, @ckrey, I had forgotten about that.

Closing as fixed.
